docs/commands/cli-v2_product-release_list.md

```diff
@@ -0,0 +1,41 @@
+## cli-v2 product-release list
+
+List all the pipelines
+
+```
+cli-v2 product-release list [flags]
+```
+
+### Examples
+
+```
+
+            cli-v2 product-release list --product <product>
+        
+```
+
+### Options
+
+```
+  -e, --end-date string               Filter by end date
```

Comment:
how does this filter works? everything `>`  than enddate? `>=`? `<`? `<=`? maybe just be clear in the description

Reply:
oh this one actually no relevant in the end, forgot to delete it

Reply:
I guess I didnt regenerated the docs